src/ocamlorg_frontend/layouts/learn_layout.eml

```diff
@@ -1,45 +1,41 @@
 type active_icon_link =
-  | Learn
+  | Overview
+  | GetStarted
+  | Platform
+  | BestPractices
   | Exercises
+  | Books
+  
 
-let sidebar_icon_link_block
+let tabs
 ~current =
-  <div class="space-y-4">
-    <a href="<%s Url.learn %>" class="flex justify-start items-center">
-      <div class="bg-primary-600 text-white rounded w-8 h-8 flex items-center justify-center">
-        <%s! Icons.book "h-5 w-5" %>
-      </div>
-      <div class="ml-3 font-semibold <%s if current != Learn then "opacity-70" else "" %>">Learn OCaml</div>
+  let link ~href ~title ~current =
+    <a href="<%s href %>" class="justify-start px-4 py-2 items-center font-semibold border-2 border-b-4 border-transparent rounded <%s if current then "no-underline text-primary-700 border-b-primary-700" else "opacity-80" %>">
+      <%s title %>
     </a>
+  in
+    <div class="bg-tab-overview text-default">
```

Comment:
Is `text-default` necessary here? It looks like we set the text color on the links, and that's usually the better place to do it.

Reply:
do you mean I move the text-default color to the nav? The `text-default` class has to be set other the text color would be white

Reply:
You explicitly set `text-primary-700` on the `a` elements when current is true. This would be a better, more precise place to set `text-default` in the case it's not current. Relying on inheritance makes the CSS harder to visualize when reading it.